java/src/test/java/ai/rapids/cudf/HostMemoryBufferTest.java

```diff
@@ -187,7 +187,7 @@ public void testFilemap() throws Exception {
   }
 
   public static void initPinnedPoolIfNeeded(long size) {
-    long available = PinnedMemoryPool.getAvailableBytes();
+    long available = PinnedMemoryPool.getTotalPoolSizeBytes();
     if (available < size) {
```

Comment:
(nit) Is this size check really needed at all?  Doesn't hurt anything though, so just a nit.

Reply:
yeah it's a little weird, but we can keep calling this function and it will shutdown/recreate it as needed, so I don't see a huge need to change it right now, plus it's pre-existing code that continues to work.